be/src/formats/parquet/file_reader.cpp

```diff
@@ -190,6 +212,34 @@ Status FileReader::_get_footer() {
     return Status::OK();
 }
 
+void FileReader::_build_split_tasks() {
+    if (!_scanner_ctx->enable_split_tasks || _scanner_ctx->split_context != nullptr ||
```

Comment:
It's better to add some comment to explain split_context and split_tasks, it's a little confusing

Reply:
OK